server/src/test/java/org/opensearch/index/shard/IndexShardTests.java

```diff
@@ -4901,10 +4906,44 @@ public void testRecordsForceMerges() throws IOException {
         closeShards(shard);
     }
 
-    private void populateSampleRemoteStoreStats(RemoteSegmentTransferTracker tracker) {
+    private void populateSampleRemoteSegmentStats(RemoteSegmentTransferTracker tracker) {
+        tracker.addUploadBytesStarted(10L);
         tracker.addUploadBytesStarted(10L);
         tracker.addUploadBytesStarted(10L);
         tracker.addUploadBytesSucceeded(10L);
         tracker.addUploadBytesFailed(10L);
     }
+
+    private void populateSampleRemoteTranslogStats(RemoteTranslogTransferTracker tracker) {
+        tracker.incrementTotalUploadsStarted();
+        tracker.incrementTotalUploadsStarted();
+        tracker.incrementTotalUploadsStarted();
```

Comment:
Has this been done intentionally? If so, why?

Reply:
Yes. This is to simulate a scenario where there already have been some uploads till now. One succeeded, one failed, and there is another which is inflight.

Reply:
Resolving this thread. Please reopen if needed.